src/js/view/controls/settings-menu.js

```diff
@@ -164,6 +164,17 @@ export function setupSubmenuListeners(settingsMenu, controlbar, viewModel, api)
         }
     }, settingsMenu);
 
+    // Visual Quality
+    model.on('change:visualQuality', (changedModel, quality) => {
+        const qualitySubMenu = settingsMenu.getSubmenu('quality');
```

Comment:
`qualitySubMenu`is not going to be defined if we're playing a video with only one level (mp4 file).

This page shows an exception in the console when trying to call `qualitySubMenu.getItems()`
http://player-pr-test-jenkins.longtailvideo.com/builds/14376/archive/test/squash/bootstrap.html?primary=html5&edition=ads&feature=api/api_setup_event_order&suite=undefined&debug=1

Just add a guard and return early.